docs/operations/runbooks/downtime_incident_management.md

```diff
@@ -0,0 +1,31 @@
+# Downtime Incident Management Runbook
+
+ Our team has agreed upon steps for handling incidents that cause our site to go offline or become unusable for users. For this document, an incident refers to one in which manage.get.gov is offline or displaying error 400/500 HTTP errors on all pages. However, for this document to apply the cause of the problem must be a critical bug in our code or one of our providers having an outage, not to be confused with a cyber security incident. This document should not be used in response to any type of cyber security incident.
+
+## Response management rules
+
+The following set of rules should be followed while an incident is in progress.
+
+- The person who first notices that the site is down is responsible for using @here and notifying in #dotgov-announce that production is down
+  - This applies to any team member, including new team members and non-developers
+- If no engineer has acknowledged the announcement within 10 minutes, whoever discovered the site was down should call each developer via the Slack DM huddle feature. If there is no response, this should escalate to a phone call.
+  - When calling, go down the [phone call list](https://docs.google.com/document/d/1k4r-1MNCfW8EXSXa-tqJQzOvJxQv0ARvHnOjjAH0LII/edit) from top to bottom until someone answers who is available to help.
+  - If this incident occurs outside of regular working hours, choosing to help is on a volunteer basis, and answering a call doesn't mean an individual is truly available to assist.
+- Once an engineer is online, they should immediately start a huddle in the #dotgov-redalert channel to begin troubleshooting
+- All available engineers should join the huddle once they see it.
+  - If downtime occurs outside of working hours, team members who are off for the day may still be pinged and called but are not required to join if unavailable to do so.
+- Uncomment the banner on get.gov, so it is transparent to users that we know about the issue on manage.get.gov
```

Comment:
I don't love leaving this in comments. I think I'd prefer we had the text in a document with clear instructions and prepared/templated language.

Reply:
We could do that but then we would need the whole banner in this separate doc (which isn't a huge issue).
@Katherine-Osos I believe you are making the text for the banner anyways, are you already using a content word document for it? Perhaps that could be used for this purpose as well? If not we could just copy your finished language and the banner and place it in a separate doc for anyone to use in an emergency.

Reply:
We could also keep the code and content for the banner in this repo somewhere.